packages/block-editor/src/hooks/index.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
The hooks are still named using `core/editor/whatever` convention. I suppose we couldn't change that?

Reply:
I don't know if these are considered public API cc @gziolo can we change these?

Reply:
Given that plugin develops might opt in for removing those hooks this is considered as a breaking change. In the past, we were able to migrate them by using hooks propagation but I'm not sure if we want to do it again.